contracts/curve/AnteCurveTVLPlungeTest.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
```

Comment:
Instead of using ERC20, you can import IERC20 instead, as it's the interface and there's nothing in this Ante Test that would require the full ERC20 contract